via/static/scripts/video_player/components/Transcript.tsx

```diff
@@ -347,9 +348,12 @@ export default function Transcript({
     return filterTranscript(transcript.segments, filter);
   }, [filter, transcript]);
 
+  useOverflowAnchor(scrollRef, element => element.querySelectorAll('li'));
```

Comment:
The browser feature that we are emulating here is called [scroll anchoring](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-anchor/Guide_to_scroll_anchoring) rather than _overflow anchoring_.

Reply:
The CSS properties that controls the behavior all have an `overflow-` prefix, which is a bit confusing. I'm guessing that was done for consistency with the existing `overflow-` properties that relate to scrolling.

Reply:
Oh, I see. Yeah, I got confused by the name of the CSS property.